autopush/db.py

```diff
@@ -256,6 +363,7 @@ def preflight_check(message, router, uaid="deadbeef00000000deadbeef00000000"):
         update_id=message_id,
         message_id=message_id,
         ttl=60,
+        expry=_expry(60),
```

Comment:
Is there some reason we can't include the i to make it a real word? It is supposed to be expiry, right?